### PR TITLE
Add a rule to disallow properties

### LINKED ISSUE #546: Disallow usage of certain properties
According to this
https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertyspelling
scss-lint supports such a feature. There is a similar linter 'no-misspelled-properties' in sass-lint but its config can include only 'extra_properties'.
What do you think about extending functionality of 'no-misspelled-properties' to support 'disabled_properties' too? Source code of this linter looks pretty simple so creating a pull request wouldn't be difficult.

### PR DESCRIPTION
**Pull request by alexeibs**
Resolves #546

DCO 1.1 Signed-off-by: Alexei Bykov alexei4203@yandex.ru

### CODE REVIEW

**Comment by DanPurdy** [comment]
Sorry to be a pain but can you change the name to no-disallowed-properties or something similar as all our other preventative or exclusion rules start with no-[something-bad]

**Comment by alexeibs** [comment]
I changed the name to 'no-disallowed-properties'

**Comment by DanPurdy** [comment]
Hi @alexeibs Sorry for not getting back to you sooner, we've been busy updating everything for our 1.6 release. Would you mind updating your branch here with the latest develop and we'll look to review this and hopefully see it come out soon!

Thanks :+1:

File: lib/rules/no-disallowed-properties.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var helpers = require('../helpers');
+
+module.exports = {
+  'name': 'no-disallowed-properties',
+  'defaults': {
+    'properties': []
+  },
+  'detect': function (ast, parser) {
```
Comment (Snugug) [inline lib/rules/no-disallowed-properties.js]: Codacy Issue found: [Expected method shorthand.](https://www.codacy.com/app/snugug/sass-lint/file/2369815718/issues/source?bid=3169912&fileBranchId=3169917#l10)

File: lib/rules/no-disallowed-properties.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
```
Comment (Snugug) [inline lib/rules/no-disallowed-properties.js]: Codacy Issue found: [Expected method shorthand.](https://www.codacy.com/app/snugug/sass-lint/file/2370059781/issues/source?bid=3169912&fileBranchId=3169917#l10)

Comment (alexeibs) [inline lib/rules/no-disallowed-properties.js]: I tried to use ES6 syntax but eslint don't like it:
`10:10  error  Parsing error: Unexpected token (`

Comment (DanPurdy) [inline lib/rules/no-disallowed-properties.js]: Hey @alexeibs ignore the comments from @Snugug with that little codacy icon at the beginning it's some tool that is enabled but doesn't seem to be functioning correctly.. It basically is ignoring the context of all of our work it seems.

**Comment by DanPurdy** [comment]
@alexeibs this looks fine, if you could add the rule with a level/status of 0 to the default rule config please https://github.com/sasstools/sass-lint/blob/develop/lib/config/sass-lint.yml 

Thanks 👍

**Comment by DanPurdy** [comment]
I've updated this in #710 with a few extra tests and the config addition, plus we can now get this merged in quicker as we're able to update the branch.

**Pull request closed without merging**